ts/nni_manager/training_service/reusable/environments/remoteEnvironmentService.ts

```diff
@@ -264,7 +264,7 @@ ${environment.command} --job_pid_file ${environment.runnerWorkingFolder}/pid \
         }
         const executor = await this.getExecutor(environment.id);
         const environmentLocalTempFolder: string =  
-            path.join(this.experimentRootDir, this.experimentId, "environment-temp")
+            path.join(this.experimentRootDir, "environment-temp")
```

Comment:
seems we don't need `cd ${environment.runnerWorkingFolder}` in L253, due to we have `cd envs/${envId}` in `trialDispatcher.ts`.

Reply:
Need to `cd ${environment.runnerWorkingFolder}` to set script root dir, other wise script will executed based on wrongly directory. This pr only fix local-temp folder, reverted env related change.